tests/test_models.py

```
@@ -32,6 +33,18 @@ def test_str_representation(self):
         assert str(subject) == "System Design"
 
 
+@pytest.mark.django_db
+class TestHumanLanguage:
+    def test_str_representation(self):
+        language = HumanLanguageFactory(name="Spanish")
+        assert str(language) == "Spanish"
+
+    def test_slug_is_unique(self):
+        HumanLanguageFactory(name="Mandarin", slug="mandarin")
+        with pytest.raises(Exception):
```

Ruff (0.15.2)

[warning] Do not assert blind exception: `Exception`

(B017)